draft-ietf-scitt-architecture.md

```diff
@@ -457,20 +462,6 @@ Verifiable Data Structure:
 SCITT supports multiple Verifiable Data Structures and Receipt formats as defined in {{-RECEIPTS}}, accommodating different Transparency Service implementations.
 {: #mybody}
 
-## Common Terminology Disambiguation
-
-This document has been developed in coordination with the COSE, OAUTH and RATS WG and uses terminology common to these working groups.
-
-This document uses the terms "Issuer", and "Subject" as described in {{RFC8392}}, however the usage is consistent with the broader interpretation of these terms in both JOSE and COSE, and the guidance in {{RFC8725}} generally applies the COSE equivalent terms with consistent semantics.
-
-The terms "Claim" as used in this document is consistent with the usage in {{RFC9711}} and {{RFC7523}}.
```

Comment:
I have dropped this, because as far I can tell it doesn't add anything to line 357, and hence dropped the two other references. Happy to add it back if someone thinks they can work it into the sentence after "defined in {{RFC8392}}".

Reply:
The reason why this text exists here was to give it a home in a place where it may become relevant again. Claim, Assertion, Attestation, Remote Attestation, Evidence, Report.... all these simply colloquial words can become named words with different meaning in some context. While the scitt architecture can live without them, without an explicit iesg review feedback to remove them, I would not remove them.